README.md

```diff
@@ -6,12 +6,118 @@ output:
+* Off shelf and ready to be used modules ('Table Downloader', 'Plot Downloader', 'File Downloader' and 'Reset Application'
+  * More modules are introduced in each new version
+* Different methods and tools to alert users and add useful information about application UI and server operations
+* Application logger with different levels and a UI tool to display and review recorded application logs
+* Application look and feel can be customized easily via 'www/periscope_style.yaml' or more advanced via 'www/css/custom.css'
+* Application can make use of JS power by customizing 'www/js/custom.js'
+
+Create and run sample app for full features demo
+
+```{r}
+periscope2::create_application(name = "demo_app", location = ".", sample_app = T, right_sidebar = T)
+shiny::runApp("./demo_app/", launch.browser = T)
+```
+
+### periscope vs periscope2
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is widely regarded and used an enterprise application development aid framework for providing the user with the ability to build a robust shiny applications and dashboards.
+
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is the direct ancestor of periscope2 in which periscope2 took most of its basic philosophy and generated applications structure.
+
+However; as periscope2 is the next generation (iteration) of [periscope](https://cran.r-project.org/web/packages/periscope/index.html) package, they differ in many aspects as follow:
+
+#### Underlying frameworks
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is depending on bootstrap 3 supported shiny packages:
+
+- [shinydashboard](https://cran.r-project.org/web/packages/shinydashboard/index.html)
+- [shinydashboard Plus](https://cran.r-project.org/web/packages/shinydashboardPlus/index.html)
+- [shinyBS](https://cran.r-project.org/web/packages/shinyBS/index.html)
+
+While periscope2 is depending only on bootstrap 4 supported package [bs4Dash](https://cran.r-project.org/web/packages/bs4Dash/index.html) to cover the same functions that are presented by [periscope](https://cran.r-project.org/web/packages/periscope/index.html) dependencies and more (i.e dashboard functionality, alerts, tool-tips, etc..) in more modernized look and feel
+
+----
+
+#### Generated applications layout
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) full generated app layout is simple as follow:
+
+- Left sidebar
+- Right sidebar
+- Header
+- Body
+
+All above parts are customizable except for header, user can only update title part in it
+
+However, periscope2 full generated app layout has the following structure:
+
+- Left sidebar
+- Right sidebar
+- Header
+- Footer
+- Body
+
+For periscope2, each part has it own UI R file that can customized and updated alone. Aslo, each part has its own dedicated alert panel except for header that has an additional alert panel for announcements framework.
```

Comment:
"that can be customized and updated" to end the first sentence. 

Second sentence: "Each part also his its own dedicated alert panel, including an announcements framework as part of the header."

Reply:
done